metron-platform/metron-elasticsearch/README.md

```diff
@@ -1,5 +1,14 @@
 # Elasticsearch in Metron
 
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Properties](#properties)
+* [Upgrading to 5.6.2](#upgrading-to-562)
+* [Type Mappings](#type-mappings)
+* [Using Metron with Elasticsearch 5.x](#using-metron-with-elasticsearch-5x)
```

Comment:
This link is broken, because the section name doesn't line up with the actual name